.golangci.yml

```diff
@@ -15,6 +15,12 @@ linters:
     - godot
     - goerr113
     - nestif
+    # TODO(christopherhein) Reenable these and fix errors
```

Comment:
Some of these have been cleaned up in #43 so I once both of these are merged we can clean this up.